antora-playbook.yml

```diff
@@ -5,6 +5,7 @@ site:
 content:
   sources:
     - url: .
+      edit_url: ~ 
```

Comment:
This actually removes the "edit page" URL thing entirely I think so it should be fixed now